package.json

```diff
@@ -51,6 +51,7 @@
     "uuid": "^8.3.2"
   },
   "dependencies": {
+    "@babel/eslint-parser": "^7.19.1",
```

Comment:
Should we move this under devDependencies?

Reply:
Good catch. Yes it should be under devDependencies